tests/test_optimizer_dryruns.py

```diff
@@ -250,7 +257,12 @@ def test_instance_type_from_cpu_memory(monkeypatch, capfd):
     assert 'Standard_D8s_v5' in stdout  # Azure, 8 vCPUs, 32 GB memory
     assert 'n2-standard-8' in stdout  # GCP, 8 vCPUs, 32 GB memory
 
-    _test_resources_launch(monkeypatch, memory=32)
+    enabled_clouds = list(clouds.CLOUD_REGISTRY.values())
+    # DO has no 32 mem instances
+    enabled_clouds.remove(clouds.CLOUD_REGISTRY['do'])
```

Comment:
Will this cause an key error on user that has no do enabled?

Reply:
i removed this filter. resolving